### PR TITLE
Add Stylee changes to API Designer

### PR DESCRIPTION
**Pull request by DimalChandrasiri**

### CODE REVIEW

File: composer/packages/theme/src/definitions/views/api-designer.less [hunk 1]
```diff
@@ -29,22 +29,68 @@
...
             }
-
-            .accordion.ui.fluid.styled {
-                padding-bottom: .5em !important;
+            .accordion {
                 margin-top: 20px;
                 border-left: 0;
                 border-right: 0;
                 border-top: 1px solid #e8e8e8;
                 border-bottom: 1px solid #e8e8e8;
```
Comment (jeradrutnam) [inline composer/packages/theme/src/definitions/views/api-designer.less]: Move all color codes to `ballerina-lang\composer\packages\theme\src\themes\default\viewsapi-designer.variables` file and use.

File: composer/packages/theme/src/definitions/views/api-designer.less [hunk 2]
```diff
@@ -29,22 +29,68 @@
...
-                    &:hover {
-                        opacity: 0.5;
+                box-shadow: none;
+                .path-op-container {
+                    display: inline;
+                    float: right;
+                    .get {
+                        border: 1px solid #2980b9;
+                        color: #2980b9;
+                        font-weight: 400;
```
Comment (jeradrutnam) [inline composer/packages/theme/src/definitions/views/api-designer.less]: Move common styles to a separate class and inherit

File: composer/packages/theme/src/definitions/views/api-designer.less [hunk 3]
```diff
@@ -29,22 +29,68 @@
...
+                    .options {
+                        border: 1px solid #43A047;
+                        color: #43A047;
                         font-weight: 400;
-                        border: 0;
                     }
                 }
+                .accordion{
+                    margin-top: 0px;
+                    .title{
```
Comment (jeradrutnam) [inline composer/packages/theme/src/definitions/views/api-designer.less]: Keep a new line in between child class name and ending of parent class properties, for a better readability and standard.

File: composer/packages/theme/src/definitions/views/api-designer.less (hunk 1 above)
Comment (DimalChandrasiri) [reply composer/packages/theme/src/definitions/views/api-designer.less]: Will be tracked with issue #13334

File: composer/packages/theme/src/definitions/views/api-designer.less (hunk 2 above)
Comment (DimalChandrasiri) [reply composer/packages/theme/src/definitions/views/api-designer.less]: Will be tracked with issue #13334

File: composer/packages/theme/src/definitions/views/api-designer.less (hunk 3 above)
Comment (DimalChandrasiri) [reply composer/packages/theme/src/definitions/views/api-designer.less]: Will be tracked with issue #13334

**Pull request merged**